multi-arch-builders/builder-common.bu

```diff
@@ -104,6 +104,7 @@ storage:
         inline: |
             [Timer]
             OnCalendar=*-*-* 05:00:00 UTC
+            OnCalendar=*-*-* 10:00:00 UTC
```

Comment:
If we do twice a day, let's have regular intervals
```suggestion
            OnCalendar=*-*-* 17:00:00 UTC
```

Reply:
Hmm... I think we hit the issue quite early in Europe, so maybe we should change the first 05:00:00 UTC to 22:00:00 instead?

Reply:
works for me